### ISSUE TITLE
Remove the DESCRIPTION label from the issue template

### ISSUE THREAD

**Issue by ykdojo** [ykdojo/defaang#49]
Since we removed the screenshot section in #43 and #46, we only have the description label in our issue templates for features and documentation.

Now that we've done that, the description label is redundant and unnecessary.

It'd be cleaner to remove it.

**Comment by Jonyango** [comment]
@ykdojo Could you please point me to where the issue template is? Can't seem to find it.

**Comment by Jonyango** [comment]
It is okay. I have found it.

**Comment by Jonyango** [comment]
@ykdojo I worked on this issue, and changes were merged with this PR #52. I didn't know I have to wait for a PR to be merged before I can push changes for a second PR. It is the last commit of that PR.

**Comment by ykdojo** [comment]
@Jonyango it's all good. We have #58 now though.

**Comment by ykdojo** [comment]
Maybe there's no way to do it? If so, I'll just close this issue.

**Comment by Preet-Sojitra** [comment]
is this still open?

**Comment by ykdojo** [comment]
@Preet-Sojitra yes it is

**Comment by Preet-Sojitra** [comment]
So what should I write in place of `Description` ? 
I guess something should be written in place of it right or if I remove it without replacing it, it won't break?

**Comment by ykdojo** [comment]
Right, that's the thing - it'll probably break if we simply remove it.

It's totally unnecessary, though.

Ideally, there is a way to remove the header without affecting everything else, but I just wasn't sure there is.

**Comment by iShibi** [comment]
Closing this as `body` is required and cannot be empty in issue templates.
